src/util/GeometryInfo.cpp

```diff
@@ -108,6 +113,30 @@ std::string boundingBoxAsWkt(const GeoPoint& lowerLeft,
   return getWKT(box);
 }
 
+// ____________________________________________________________________________
+template <detail::constexpr_str_cat_impl::ConstexprString suffix>
+constexpr std::string_view addSfPrefix() {
+  return constexprStrCat<SF_PREFIX, suffix>();
+}
```

Comment:
Not sure how reliable the coverage tool is currently (or for constexpr things specifically), but I'd appreciate a very basic unit test just to please the tool and prevent future typos.